problemsets-mat223/linearalbegra.tex

```diff
@@ -1970,8 +1970,29 @@ \section*{Dot Product}
 				vector whose distance from $\vec u$ is 1 can have is $\sqrt{6} - 1$, 
 				which is greater than 1. 
 			\end{solution}
-		\item Suppose $\vec y$ is a unit vector and the distance between $\vec y$ and
-			$\vec u$ is $2$.  What is the angle between $\vec y$ and $\vec u$?
+		\item Suppose $\vec y$ is a unit vector and the distance between $\vec y$ 
+			and	$\vec u$ is $2$. What is the angle between $\vec y$ and $\vec u$?
+			\begin{solution}
+				The angle between them is $\arccos\left(-\frac{3}{2\sqrt{6}}\right)$. 
+
+				\
+
+				From the theorem above, we know that $2=\|\vec u-\vec y\|=\sqrt{(\vec u-\vec y)\cdot(\vec u-\vec y)}$.
```

Comment:
Use `\norm`

Reply:
You can just say, "By assumption 2=|u-y| and so 4=|u-y|^2=(u-y)*(u-y) ..."

Reply:
Cool. Do you want me to edit all of your other uses of \| ... \| to use \norm as well? 

And okay, I'll cut that down a bit. I thought it was a good idea to mention the theorem specifically, but without that it can be shorter for sure.